ignite/metrics/mean_pairwise_distance.py

```diff
@@ -26,6 +26,32 @@ class MeanPairwiseDistance(Metric):
         device: specifies which device updates are accumulated on. Setting the
             metric's device to be the same as your ``update`` arguments ensures the ``update`` method is
             non-blocking. By default, CPU.
+
+    Examples:
+        To use with ``Engine`` and ``process_function``, simply attach the metric instance to the engine.
+        The output of the engine's ``process_function`` needs to be in the format of
+        ``(y_pred, y)`` or ``{'y_pred': y_pred, 'y': y, ...}``. If not, ``output_tranform`` can be added
+        to the metric to transform the output into the form expected by the metric.
+
+        ``y_pred`` and ``y`` should have the same shape.
+
+        .. testcode::
+
+            metric = MeanPairwiseDistance(p=4)
+            metric.attach(default_evaluator, 'l4_distance')
```

Comment:
```suggestion
            metric.attach(default_evaluator, 'mpd')
```
or anything else but